BadHabbit/swiftgen.yml

```diff
@@ -0,0 +1,86 @@
+## Note: all of the config entries below are just examples with placeholders. Be sure to edit and adjust to your needs when uncommenting.
+
+## In case your config entries all use a common input/output parent directory, you can specify those here.
+##   Every input/output paths in the rest of the config will then be expressed relative to these.
+##   Those two top-level keys are optional and default to "." (the directory of the config file).
+input_dir: BadHabbit
+output_dir: BadHabbit
+
+
+## Generate constants for your localized strings.
+##   Be sure that SwiftGen only parses ONE locale (typically Base.lproj, or en.lproj, or whichever your development region is); otherwise it will generate the same keys multiple times.
+##   SwiftGen will parse all `.strings` files found in that folder.
+strings:
+  inputs:
+    - SupportFiles/Base.lproj
+  outputs:
+    - templateName: structured-swift5
+      output: Strings+Generated.swift
+
+
+## Generate constants for your Assets Catalogs, including constants for images, colors, ARKit resources, etc.
+##   This example also shows how to provide additional parameters to your template to customize the output.
+##   - Especially the `forceProvidesNamespaces: true` param forces to create sub-namespace for each folder/group used in your Asset Catalogs, even the ones without "Provides Namespace". Without this param, SwiftGen only generates sub-namespaces for folders/groups which have the "Provides Namespace" box checked in the Inspector pane.
+##   - To know which params are supported for a template, use `swiftgen template doc xcassets swift5` to open the template documentation on GitHub.
+xcassets:
+  inputs:
+    - SupportFiles/Assets.xcassets
+  outputs:
+    - templateName: swift5
+      params:
+        forceProvidesNamespaces: true
+      output: XCAssets+Generated.swift
+
+
+## Generate constants for your storyboards and XIBs.
```

Comment:
Clean this file. Commented lines make it's hard to read this config

Reply:
Unused code removed